### PR TITLE
Fix: Fixes issue where bullet won't get the right range

### PR DESCRIPTION
**Pull request by Golodhros**
## Description
Fixes problem with bullet not getting ranges properly

## Motivation and Context
Bug fix

## How Has This Been Tested?
Ran tests
Checked manually with extended example

## Screenshots (if appropriate):
Tutorial__Bullet_Chart

## Types of changes

-   [ ] Refactor (changes the way we code something without changing its functionality)
-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [ ] Latest master code has been merged into this branch
-   [ ] No commented out code (if required, place // TODO above with explanation)
-   [ ] No linting issues
-   [ ] Build is successful
-   [ ] Code follows the [API Guidelines](http://britecharts.github.io/britecharts/topics-index.html#toc5__anchor)
-   [ ] Updated the documentation
-   [ ] Added tests to cover changes
-   [ ] All new and existing tests passed

### CODE REVIEW

File: test/fixtures/bulletChartDataBuilder.js [hunk 1]
```diff
@@ -35,6 +35,50 @@ define(function(require) {
             return new this.Klass(attributes);
         };
 
+        this.withManyCPUData = function () {
```
Comment (DalerAsrorov) [inline test/fixtures/bulletChartDataBuilder.js:38]: for consistency sake, let's have a json file for this dataset as well

**Review by DalerAsrorov — APPROVED** [review]

File: test/fixtures/bulletChartDataBuilder.js (hunk 1 above)
Comment (Golodhros) [reply test/fixtures/bulletChartDataBuilder.js:38]: I am actually thinking of dumping the JSON files.

They add a level of indirection that is a pain, especially when dealing with tests and examples. It is true that there are a couple of datasets (line and area chart) that are large, but most of the time is not a big deal and our IDE's collapsing features help a lot.

what do you think?

Comment (DalerAsrorov) [reply test/fixtures/bulletChartDataBuilder.js:38]: Sounds good to me. If we were using TypeScript dataset types could be highlighted if we were to place a wrong key or value type, so I would go with not using json files as well.

Comment (Golodhros) [reply test/fixtures/bulletChartDataBuilder.js:38]: Cool, I will go ahead and refactor that on the RC

**Pull request merged**